.github/workflows/download_count.yaml

```diff
@@ -0,0 +1,47 @@
+name: Update Download Count
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  update-downloads:
+    name: Update Download Count
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get download count
+        run: |
+          # check downloads
+          downloads=$(curl https://api.npmjs.org/downloads/point/1970-01-01:$(date -d '+1 day' +"%Y-%m-%d")/colorjs.io | jq -r .downloads)
+          echo "Latest download count: $downloads"
+          # into millions
+          downloads=$((downloads / 1000000))
```

Comment:
Does this also round? I'm not very familiar with these and can't tell where the number of downloads is rounded.

Reply:
This code will just truncate the number instead of rounding to the nearest million (eg. 5,999,999 downloads would become be 5 million). I can make it round instead, if preferred

Reply:
Truncating is probably fine, rounding might be better if easy.

Reply:
I just changed it to round instead of truncate, but I've also just realized that the README says "_over_ 10 million total downloads," so it might be a bit misleading to round? I'm not sure how important that is, but figured it's worth mentioning